tests/vm_console_proxy_test.go

```diff
@@ -0,0 +1,135 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+)
+
+var _ = Describe("VM Console Proxy Operand", func() {
+	var (
+		clusterRoleResource        testResource
+		clusterRoleBindingResource testResource
+		serviceAccountResource     testResource
+		serviceResource            testResource
+		deploymentResource         testResource
+		configMapResource          testResource
+	)
+
+	BeforeEach(func() {
+		expectedLabels := expectedLabelsFor("vm-console-proxy", "vm-console-proxy")
+		clusterRoleResource = testResource{
+			Name:           "vm-console-proxy",
+			Resource:       &rbac.ClusterRole{},
+			ExpectedLabels: expectedLabels,
+		}
+		clusterRoleBindingResource = testResource{
+			Name:           "vm-console-proxy",
+			Resource:       &rbac.ClusterRoleBinding{},
+			ExpectedLabels: expectedLabels,
+		}
+		serviceAccountResource = testResource{
+			Name:           "vm-console-proxy",
+			Namespace:      strategy.GetVmConsoleProxyNamespace(),
+			Resource:       &core.ServiceAccount{},
+			ExpectedLabels: expectedLabels,
+		}
+		serviceResource = testResource{
+			Name:           "vm-console-proxy",
+			Namespace:      strategy.GetVmConsoleProxyNamespace(),
+			Resource:       &core.Service{},
+			ExpectedLabels: expectedLabels,
+		}
+		deploymentResource = testResource{
+			Name:           "vm-console-proxy",
+			Namespace:      strategy.GetVmConsoleProxyNamespace(),
+			Resource:       &apps.Deployment{},
+			ExpectedLabels: expectedLabels,
+		}
+		configMapResource = testResource{
+			Name:           "vm-console-proxy",
+			Namespace:      strategy.GetVmConsoleProxyNamespace(),
+			Resource:       &core.ConfigMap{},
+			ExpectedLabels: expectedLabels,
+		}
+
+		waitUntilDeployed()
+	})
+
+	Context("Resource creation", func() {
+		DescribeTable("created cluster resource", func(res *testResource) {
+			resource := res.NewResource()
+			err := apiClient.Get(ctx, res.GetKey(), resource)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(hasOwnerAnnotations(resource.GetAnnotations())).To(BeTrue(), "owner annotations are missing")
+		},
+			Entry("[test_id:1000] cluster role", &clusterRoleResource),
```

Comment:
These `test_id` are created by QE, and correspond to test plans in their system.
For now can you write `TODO` here, until the IDs are created?

Reply:
Okay, who will responsible then to monitor the creation of the test plans and update them here? Will be there a ticket to update IDs?

Reply:
You can ask QE to create the test IDs.